doc/trace-api.md

```diff
@@ -32,11 +9,11 @@ A `TraceApi` instance provides functions that facilitate the following:
 
 In addition to the above, `TraceApi` also provides a number of well-known label keys and constants through its `labels` and `constants` fields respectively.
 
-#### Trace Spans
+## Trace Spans
 
 These functions provide the capability to create trace spans, add labels to them, and close them.
 
-* `TraceApi#api.runInRootSpan(options, fn)`
+* `TraceApi#runInRootSpan(options, fn)`
```

Comment:
We should probably mention here that the `endSpan` method needs to be manually called.  Otherwise, the trace won't get published.